packages/wonder-blocks-data/src/util/types.js

```diff
@@ -19,6 +26,9 @@ export type Result<TData: ValidData> =
           status: "aborted",
       |};
 
+/**
+ * A cache entry for a fulfilled request response.
+ */
 export type CacheEntry<TData: ValidData> =
```

Comment:
Ideally, `ResponseCacheEntry` or something, but naming things is hard and I decided to leave existing types as they are for now.

Reply:
Naming is fixed up in https://github.com/Khan/wonder-blocks/pull/1341?w=1